src/Model/ModelManager.php

```diff
@@ -348,9 +348,9 @@ public function getIdentifierValues($entity)
                 continue;
             }
 
-            $metadata = $this->getMetadata(ClassUtils::getClass($value));
+            $identifierMetadata = $this->getMetadata(ClassUtils::getClass($value));
```

Comment:
This is just a variable rename?

Reply:
Yes

Reply:
The goal is to avoid reusing the same variable to hold 2 different objects 👍